examples/postgres/execute/main.ml

```diff
@@ -15,30 +15,30 @@ let () =
   (* Start the database connection pool *)
   let* db =
     let config =
-      Silo_postgres.config
+      Silo.config
         ~connections:5
         ~connection_string:
           "postgresql://postgres:postgres@localhost:6432/postgres?sslmode=disabled"
     in
 
-    match Silo_postgres.connect ~config with
+    match Silo.connect ~config with
     | Ok c -> Ok c
     | Error (`Msg e) -> Error e
   in
 
   (* Fetch the user and return the user to a variable *)
   let* rows_affected =
-    Silo_postgres.execute
+    Silo.execute
       db
       ~params:
         [
-          Silo_postgres.Params.String "Emil";
-          Silo_postgres.Params.Bool true;
-          Silo_postgres.Params.String "Danza";
-          Silo_postgres.Params.Number 1;
-          Silo_postgres.Params.Number 1;
-          Silo_postgres.Params.Float 1.1;
-          Silo_postgres.Params.StringArray ["Danza"];
+          Silo.Params.String "Emil";
+          Silo.Params.Bool true;
+          Silo.Params.String "Danza";
+          Silo.Params.Number 1;
+          Silo.Params.Number 1;
+          Silo.Params.Float 1.1;
+          Silo.Params.StringArray ["Danza"];
```

Comment:
```suggestion
          string "Emil";
          bool true;
          string "Danza";
          number 1;
          number 1;
          float 1.1;
          string_array ["Danza"];
```

making these functions instead of constructors means you can change the internal type representation later without having to refactor client code.

Reply:
Yeah I agree with this. Way nicer. good feedback! :D